junit5-jupiter-starter-bazel/WORKSPACE

```diff
@@ -1,13 +1,48 @@
-load(":junit5.bzl", "junit_jupiter_java_repositories", "junit_platform_java_repositories")
-
 JUNIT_JUPITER_VERSION = "5.9.1"
-
 JUNIT_PLATFORM_VERSION = "1.9.1"
 
-junit_jupiter_java_repositories(
-    version = JUNIT_JUPITER_VERSION,
+RULES_JVM_EXTERNAL_VERSION = "4.2"
+RULES_JVM_EXTERNAL_SHA = "cd1a77b7b02e8e008439ca76fd34f5b07aecb8c752961f9640dea15e9e5ba1ca"
+
+CONTRIB_RULES_JVM_VERSION = "0.9.0"
+CONTRIB_RULES_JVM_SHA = "548f0583192ff79c317789b03b882a7be9b1325eb5d3da5d7fdcc4b7ca69d543"
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_VERSION,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_VERSION,
+)
+http_archive(
+    name = "contrib_rules_jvm",
+    sha256 = CONTRIB_RULES_JVM_SHA,
+    strip_prefix = "rules_jvm-%s" % CONTRIB_RULES_JVM_VERSION,
+    url = "https://github.com/bazel-contrib/rules_jvm/archive/refs/tags/v%s.tar.gz" % CONTRIB_RULES_JVM_VERSION,
```

Comment:
I'd personally move this just above `contrib_rules_jvm_deps` so that downloads and configurations are grouped together. (The same goes for `CONTRIB_RULES_JVM_*` constants)

Reply:
Done in 2b27296d0fce1a7dc32167f4dedb88db25329f82.